converter/src/test/java/gov/cms/qpp/conversion/validate/CheckerTest.java

```diff
@@ -709,6 +710,28 @@ void testUnhappyValueIsNotEmptyAsNull() throws Exception {
 				.containsExactly(ERROR_MESSAGE);
 	}
 
+	@Test
+	void testIsValidDateWithValidDate() {
```

Comment:
We should add other tests that tests values that include dashes, slashes, and other formats.

Reply:
I added this. Will push soon.

Reply:
Tests were added in `ReportingParametersActValidatorTest`. I felt it was a more appropriate place than in `Checker`